clusterloader2/testing/density/config.yaml

```diff
@@ -1,5 +1,13 @@
+#Constants
+{{$NODES_PER_NAMESPACE := 100}}
+{{$PODS_PER_NODE := 30}}
+#Variables
+{{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
+{{$namespaces := DivideInt (SubtractInt .Nodes 1) $NODES_PER_NAMESPACE | AddInt 1}}
```

Comment:
(IIUC) Why are you subtracting 1 from #nodes?

Reply:
To not create additional namespace when we have `nodes` == `nodes_per_namespace` (* `n`). This is a handling for this one specific case.

Reply:
SG